topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorTestUtil.java

```diff
@@ -130,4 +138,17 @@ static <T> T waitUntilCondition(Resource<T> resource, Predicate<T> condition) {
     static KafkaTopic findKafkaTopicByName(List<KafkaTopic> topics, String name) {
         return topics.stream().filter(kt -> kt.getMetadata().getName().equals(name)).findFirst().orElse(null);
     }
+    
+    static String stringFromFile(File filePath) {
```

Comment:
Why not just do something like `new String(Files.readAllBytes(Paths.get(fileName)))`?

Reply:
`Files.lines` works well with small (this case) and large text files, so I think it's better for a general purpose util method. I just did some improvements.